README.md

```diff
@@ -1,30 +1,160 @@
 # matrix-appservice-slack
 
-[![Docker Automated build](https://img.shields.io/docker/cloud/build/matrixdotorg/matrix-appservice-slack.svg)](https://hub.docker.com/r/matrixdotorg/matrix-appservice-slack)
-[![#slack:half-shot.uk](https://img.shields.io/matrix/slack:half-shot.uk.svg?server_fqdn=chaotic.half-shot.uk&label=%23slack:half-shot.uk&logo=matrix)](https://matrix.to/#/#slack:half-shot.uk)
-[![Documentation Status](https://readthedocs.org/projects/matrix-appservice-slack/badge/?version=latest)](https://matrix-appservice-slack.readthedocs.io/en/latest/?badge=latest)
+This is **a fork of [matrix-appservice-slack](https://github.com/matrix-org/matrix-appservice-slack)**, which adds some new features and fixes several issues that have been longstanding upstream. This fork is a **drop-in replacement** for the upstream bridge, see [Usage](#usage) for deployment instructions. 
 
-A bridge that connects [Matrix](https://matrix.org) and [Slack](https://slack.com). 
-The bridge is considered **stable** and mature for use in production
-environments.
+The Slack bridge of the [community.wordpress.org](https://community.wordpress.org) homeserver is running this fork.
 
-![Screenshot](screenshot.png)
+## Features
 
-## Requirements
+In addition to all features of the upstream bridge, this fork adds the following:
 
-Hosting this bridge requires you to have a Matrix homeserver. In order to
-connect a Slack Workspace to your bridge, you will need permission to add bots
-to it. You will also need Node.JS 16+ or Docker on your system.
+- Matrix puppeting
+  - Disabled by default, see [Matrix puppeting](#matrix-puppeting).
```

Comment:
Is that not disabled by default in upstream?

Reply:
Upstream doesn't have this feature, it only has Slack puppeting. This "Matrix puppeting" is about the bridge posting as the Matrix user, instead of as a "ghost" (`@slack_*:example.org`).